atc/exec/get_step.go

```diff
@@ -189,6 +189,7 @@ func (step *GetStep) Run(ctx context.Context, state RunState) error {
 	if err != nil {
 		return err
 	}
+	step.delegate.Stderr().Write([]byte(fmt.Sprintf("Chosen worker %s: %s\n", chosenWorker.Name(), chosenWorker.Description())))
```

Comment:
what is the reasoning behind using `stderr` vs `stdout` here? Thx.

Reply:
There are some discussions in PR #4398 between @vito and me. I believe Concourse generated logs (sys log) should go to stderr. Going forward, @vito is thinking to add a separate log event channel to print Concourse generated logs.